pkg/command/root.go

```diff
@@ -209,6 +218,11 @@ func isStandalonePluginCommand(cmd *cobra.Command) bool {
 	return exists && scope == common.PluginScopeStandalone
 }
 
+func isPluginCommand(cmd *cobra.Command) bool {
+	t, exists := cmd.Annotations["type"]
+	return exists && t == "plugin"
```

Comment:
Nit: could you make "plugin" a constant?  That way we can ask the IDE to show us where that annotation is set.  Using a plain "plugin" string makes it a little more difficult to trace this logic.